lib/instances/CoreGui.lua

```diff
@@ -1,3 +1,7 @@
 local BaseInstance = import("./BaseInstance")
+local CoreGui = BaseInstance:extend("CoreGui")
+local InstanceProperty = import("../InstanceProperty")
 
-return BaseInstance:extend("CoreGui")
\ No newline at end of file
+CoreGui.properties.RobloxGui = InstanceProperty.normal({})
```

Comment:
Instead of creating a property like this, can we add an initializer for `CoreGui` that creates an actual `ScreenGui` instance named `RobloxGui` as a child?